site/app/templates/course/CourseMaterials.twig

```diff
@@ -1,5 +1,6 @@
 {% import _self as self %}
-<script type="text/javascript" language="javascript" src="js/jquery-ui-timepicker-addon.js"></script>
+<link rel="stylesheet" href="css/flatpickr.min.css">
+<script type="text/javascript" language="javascript" src="js/flatpickr.js"></script>
```

Comment:
see comment about script/link tags

Reply:
I think I have resolved both. Thank you for you advice.